
public class Exception_Handling {

	public static void main(String[] args) {
		
		//System.out.println("Hii");

		try {
			System.out.println("OO");
		
			System.out.println(10/0);
		}
		
		catch(ArithmeticException e)
		
		{
			System.out.println(5/0);
		}
		
		//System.out.println("hello");

		
		finally 
		{
			System.out.println("ooo");
		}
						
		
		/*try 
		{
			System.out.println(1);
			System.out.println(2);
			System.out.println(3);
		}
		catch(ArithmeticException e)
		{
			System.out.println(4);
			
		}
		
		finally {System.out.println(5);}
		
		System.out.println(6);*/
		
		
		
	}

}


		

